workflow/tests/base_results/results.csv

```diff
@@ -288,7 +288,7 @@ base-misc-usage-multiplier.xml,49.6,49.6,61.3,0.0,2.25,4.95,0.0,0.0,0.0,0.36,3.5
 base-pv.xml,34.89,8.02,15.01,0.0,0.0,0.0,0.0,0.0,0.0,0.39,3.42,0.65,9.15,0.0,0.0,4.51,0.0,0.33,0.0,0.0,0.0,0.0,2.22,0.0,0.0,0.32,0.37,1.51,1.53,0.0,2.11,8.37,0.0,0.0,0.0,0.0,0.0,0.0,-26.86,0.0,15.01,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,14.19,11.03,9.22,0.61,0.0,0.0,0.0,0.01,6573.0,6641.0,23.39,20.81,0.0,3.84,3.94,0.54,4.1,0.93,-0.25,0.0,0.0,3.18,-0.1,5.42,0.0,0.49,0.0,3.43,-11.43,0.0,-0.01,-0.44,-0.02,3.78,-0.15,8.98,0.0,0.0,-5.86,-0.04,-1.15,-6.67,-0.11,0.0,2.89,9.86,1354.0,998.0,11562.0,2653.0
 base-schedules-smooth.xml,34.89,34.89,15.0,0.0,0.0,0.0,0.0,0.0,0.0,0.39,3.42,0.65,9.14,0.0,0.0,4.51,0.0,0.33,0.0,0.0,0.0,0.0,2.22,0.0,0.0,0.32,0.37,1.51,1.53,0.0,2.12,8.38,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,15.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,14.18,11.04,9.21,0.61,0.0,0.0,0.0,0.01,6570.0,6637.0,23.4,20.79,0.0,3.84,3.94,0.54,4.1,0.93,-0.25,0.0,0.0,3.18,-0.1,5.42,0.0,0.49,0.0,3.43,-11.44,0.0,-0.01,-0.44,-0.02,3.78,-0.15,8.98,0.0,0.0,-5.86,-0.04,-1.15,-6.67,-0.11,0.0,2.9,9.86,1354.0,998.0,11558.0,2652.0
-base-schedules-stochastic-vacancy.xml,29.73,29.73,18.19,0.0,0.0,0.0,0.0,0.0,0.0,0.47,3.49,0.67,7.42,0.0,0.0,3.62,0.0,0.26,0.0,0.0,0.0,0.0,2.22,0.0,0.0,0.27,0.3,1.26,1.26,0.0,1.7,6.78,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,18.19,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,17.2,11.35,7.36,0.61,0.0,0.0,0.0,0.01,5363.0,5770.0,21.23,20.82,0.0,3.79,3.9,0.54,3.79,0.92,-0.36,0.0,0.0,2.8,-0.09,5.53,0.0,0.07,0.0,4.09,-7.88,0.0,-0.03,-0.46,-0.02,3.74,-0.16,8.94,0.0,0.0,-5.89,-0.04,-1.19,-6.78,-0.02,0.0,2.96,10.33,1141.0,884.0,9206.0,2112.0
+base-schedules-stochastic-vacancy.xml,35.04,35.04,17.28,0.0,0.0,0.0,0.0,0.0,0.0,0.45,3.68,0.72,8.9,0.0,0.0,4.51,0.0,0.33,0.0,0.0,0.0,0.0,2.22,0.0,0.0,0.32,0.37,1.51,1.53,0.0,2.12,8.38,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,17.28,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,16.33,12.18,8.95,0.61,0.0,0.0,0.0,0.01,6259.0,6518.0,21.01,21.28,0.0,3.8,3.91,0.54,3.91,0.92,-0.31,0.0,0.0,3.0,-0.07,5.54,0.0,0.07,0.0,3.9,-8.98,0.0,-0.09,-0.51,-0.03,3.55,-0.17,8.8,0.0,0.0,-6.09,-0.03,-1.24,-7.1,-0.02,0.0,3.13,11.99,1355.0,998.0,11288.0,2590.0
```

Comment:
Hmm, apparently the set_vacancy code does need to be later. Curious.

Reply:
@shorowit It's down below because we need to calculate design levels (e.g., https://github.com/NREL/OpenStudio-HPXML/blob/bb8d2e2a1e071b833f49a00a239ccd07995cb734/HPXMLtoOpenStudio/resources/misc_loads.rb#L19) *before* schedule values are zeroed out.